packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -285,11 +295,19 @@ export interface CallAdapterCallOperations {
     // @beta
     resumeCall(): Promise<void>;
     // @beta
+    selectUploadedBackground(backgroundImage: BackgroundImage): void;
+    // @beta
     sendDtmfTone(dtmfTone: DtmfTone_2): Promise<void>;
+    // @beta
+    setCustomBackgroundImages(backgroundImages: BackgroundImage[]): void;
     startCamera(options?: VideoStreamOptions): Promise<void>;
     startScreenShare(): Promise<void>;
+    // @beta
+    startVideoBackgroundEffect(videoBackgroundEffect: VideoBackgroundBlurEffect | VideoBackgroundReplacementEffect): Promise<void>;
```

Comment:
Can we use Calling's types here instead of defining our own?
![image](https://user-images.githubusercontent.com/2684369/221318764-11f253bd-87a9-4c92-858e-f3bc2fc6dec0.png)